tests/lib/rules/no-constant-binary-expression.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * @fileoverview Tests for no-constant-binary-expression rule.
+ * @author Jordan Eldredge <https://jordaneldredge.com>
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-constant-binary-expression");
+const { RuleTester } = require("../../../lib/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2021, ecmaFeatures: { jsx: true } } });
+
+const invalid = [
```

Comment:
Can you move this down to where the variable is used? We generally have valid cases first and invalid second in the test files.